lego/apps/users/views/memberships.py

```diff
@@ -33,3 +39,21 @@ def get_queryset(self):
     def create(self, request, *args, **kwargs):
         request.data["abakus_group"] = kwargs["group_pk"]
         return super(MembershipViewSet, self).create(request, *args, **kwargs)
+
+    def update(self, request: HttpRequest, *args: Any, **kwargs: Any) -> Response:
+        """Create a new membership history object when a membership is updated"""
+
+        user = User.objects.get(pk=request.data["membership"]["id"])
+        group = AbakusGroup.objects.get(pk=request.data["membership"]["abakus_group"])
+        start_date = datetime.strptime(
+            request.data["membership"]["created_at"], "%Y-%m-%dT%H:%M:%S.%fZ"
+        )
```

Comment:
Do `serializer.is_valid()`  and use the serialized value in the `create` below and you can throw away this.